bitnami/gitlab-runner/templates/configmap.yaml

```diff
@@ -0,0 +1,287 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if not .Values.existingConfigMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Comment:
It seems you're using this ConfigMap to contain both configuration and scripts (entrypoint, healthchek, etc.). I'd split it into 2 configmaps

Reply:
I didn't separate it to make it consistent with upstream. But I believe it doesn't hurt much separating them.